src/PopEngine/index.ts

```diff
@@ -220,7 +222,6 @@ export class PopEngine {
           this.setState(p, PopStateType.HIDDEN, p.opts, null, false);
 
           if (!!popOver) {
-            document.body.removeChild(popOver);
```

Comment:
I actually had code that would check if the popover was part of the body, but then decided not to add in the logic unless we get more exceptions to show up, might be overkill.